src/libponyrt/sched/cpu.c

```diff
@@ -12,6 +12,7 @@
 #elif defined(PLATFORM_IS_MACOSX)
   #include <unistd.h>
   #include <mach/mach.h>
+  #include <mach/mach_time.h>
```

Comment:
I believe this is BigSur only and doesn't exist prior. Am I mistaken?

Reply:
I have it on Mojave. The reason this header needs to be included now is related to `mach_absolute_time` [called below](https://github.com/ponylang/ponyc/blob/a9523f41e60cb04c0bd2042a65e8795cabc21b0b/src/libponyrt/sched/cpu.c#L385-L387) which is only called on ARM platforms, and lives in that specific header. The include could probably be guarded by `#if defined PLATFORM_IS_ARM`.

Reply:
Ah, ok.